sites/gcweb-menu/gcweb-menu-en.html

```diff
@@ -0,0 +1,45 @@
+---
+{
+	"title": "GCWeb Menu",
+	"language": "en",
+	"altLangPage": "gcweb-menu-fr.html",
+	"secondlevel": false,
+	"dateModified": "2022-05-12",
+	"share": "true"
+}
+---
+<div class="wb-prettify all-pre hide"></div>
+
+<p>The <a href="#v1">GCWeb menu version one (v1)</a> is deprecated but still supported.</p>
+
+<h2>Version 2</h2>
```

Comment:
Je décrirerais le changement majeur. par example:
```suggestion
<h2>Version 2</h2>
<p>Need to be used along with the <a href="../header/header-en.html">version 2 of site header</a></p>
<p>List of change since version 1: Removal of the <code>&lt;div class="container"&gt;</code>.</p>
```